kalite/main/api_resources.py

```diff
@@ -71,110 +71,6 @@ class Meta:
         authorization = UserObjectsOnlyAuthorization()
 
 
-class Video:
```

Comment:
This model was used for the tastypie Resource endpoint for Video item data. This has now been subsumed under the very similar code for 'Content'.